src/geo/cartodb-layer-group-view-base.js

```diff
@@ -20,12 +21,20 @@ CartoDBLayerGroupViewBase.prototype = {
     this._clearInteraction();
 
     this.model.forEachGroupedLayer(function (layerModel, layerIndex) {
-      if (layerModel.isVisible()) {
+      if (this._isIteractionEnabledForLayer(layerModel)) {
         this._enableInteraction(layerIndex);
       }
     }, this);
   },
 
+  _isIteractionEnabledForLayer: function (layerModel) {
+    if (!layerModel.isVisible()) {
+      return false;
+    }
+    // Builder needs interactivity when the layer has no popup/infowindows to allow feature edition.
+    return layerModel.isInteractive() || this._mapModel.isFeatureInteractivityEnabled();
```

Comment:
The second condition has little to do with the layer as the name of this method suggests so I would extract it from this method. It might also make sense to include the `isVisible` part inside of `isInteractive`? Perhaps not...

Reply:
yep, the name is not the best... maybe `_shouldEnableInteraction(layer)`  ?